de_DE.csv

```diff
@@ -8933,7 +8934,7 @@ You may need more permissions to access this category.,Ihre Berechtigungen reich
 You may need more permissions to access this product.,Ihre Berechtigungen reichen nicht aus um diesen Artikel anzusehen.
 You moved the category,Sie haben die Kategorie verschoben.
 You must buy at least %1 of these per purchase.,Sie müssen mindestens %1 dieser Artikel pro Bestellung kaufen.
-You must confirm your %customer_email email before you can sign in (link is only valid once):,"Sie müssen Ihre %customer_email bestätigen, bevor Sie sich einloggen (Link ist nur einmal gültig):"
+You must confirm your %customer_email email before you can sign in (link is only valid once):,"Sie müssen Ihre %customer_email Email bestätigen, bevor Sie sich einloggen (Link ist nur einmal gültig):"
```

Comment:
The email address, for example "You must confirm your ab@c.de email before ...".